src/devices/tuya.ts

```diff
@@ -4257,6 +4257,28 @@ const definitions: DefinitionWithExtend[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_2rvvqjoa']),
```

Comment:
Please merge with `TS0601_cover_10` and use `tuya.whitelabel` for proper detection (see examples in this file)

Reply:
Missed that, let's keep it like this.